### PR TITLE
Add a PR check that detects unused images

### PR DESCRIPTION
**Pull request by sleshchenko**
> Read our [Contribution guide](https://github.com/eclipse/che-docs/blob/master/CONTRIBUTING.adoc) before submitting a PR.

### What does this PR do?
This idea came after I made my PR on che-docs and forgot to remove images after cleaning up the article(thanks reviewers for catching it up).
So, I've spent some time on my Hack&Hustle and added a script that detects unused images and it's reported via dedicated PR check.
:notebook: 
```
$ shellcheck detect-unused-images.sh
No issues detected!
```
I would say the next step is making this PR check required.

Dear reviewer, here you can take a look at how this PR check works 
https://github.com/eclipse/che-docs/pull/1755/checks?check_run_id=1576970041

Screenshot_20201218_154054
Screenshot_20201218_154114

^ :crossed_fingers: hope I did not miss anything and these images are really unused.

### What issues does this PR fix or reference?
N/A

### Specify the version of the product this PR applies to.
N/A

### PR Checklist

As the author of this Pull Request I made sure that:

- [ ] `vale` has been run successfully against the PR branch
- [ ] Link checker has been run successfully against the PR branch
- [ ] Documentation describes a scenario that is already covered by QE tests, otherwise an issue has been created and acknowledged by Che QE team
- [ ] Changed article references are updated where they are used (or a redirect has been set up on the docs side):
    - [ ] Dashboard [branding.json](https://github.com/eclipse/che-dashboard/blob/master/src/components/branding/branding.json)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)

### CODE REVIEW

File: tools/detect-unused-images.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        continue
+    fi
+
+    readarray -d '' images < <(find "images" -name '*.png' -print0)
+    for image in "${images[@]}"
+    do
+        #`../` instead of `images/` is used in the documentation references
+        image=${image#"images/"}
+        image_with_che_context="${image/che/{project-context\}}"
+        image_with_crw_context="${image/crw/{project-context\}}"
```
Comment (sleshchenko) [inline tools/detect-unused-images.sh]: I'm not sure that master should contain crw-related images.
Currently there is `keycloak/crw-keycloak-username-readonly-theme.png`

I can rework it in way so each branch defines project context value.

Comment (themr0c) [reply tools/detect-unused-images.sh]: There should be nothing mentioning crw here... Historical mistakes. It will help for the cleanup.

**Review by themr0c — APPROVED** [review]

File: tools/detect-unused-images.sh (hunk 1 above)
Comment (sleshchenko) [reply tools/detect-unused-images.sh]: Let me then update the PR

**Review by themr0c — APPROVED** [review]

**Comment by themr0c** [comment]
Uasge issue in hosted che:
```
tools/detect-unused-images.sh: line 14: find: command not found
All images have references in their the modules.
```

**Pull request merged**